docs/interpreter/beam.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: "Beam Interpreter"
+description: ""
+group: interpreter
+---
```

Comment:
This front formatter info will be used to search Zeppelin docs in website.(this feature is newly added by #1266 ) So it would be better we update this info like below like other `docs/interpreter/*.md` files.

```
layout: page
title: Beam interpreter in Apache Zeppelin
description: Apache Beam is an open source, unified programming model that you can use to create a data processing pipeline.
group: interpreter
```